docs/reference/composites.rst

```diff
@@ -258,3 +258,69 @@ Methods
    VirtualGraphComposite.sample
    VirtualGraphComposite.sample_ising
    VirtualGraphComposite.sample_qubo
+
+Reverse
+=======
+
+Composites that do batch reverse annealing. Can accept batch initial states or a set
+of schedules.
+
+BatchReverseComposite
+---------------------
```

Comment:
Please consider starting all the reverse-anneal composites with "Reverse" and then the base for expansion; so something like,
ReverseBatchInitialStates
ReverseBatchSchedules